### PR TITLE
Create README.md

### PR DESCRIPTION
**Pull request by jopopcorn**
* Create Euphony Speaker's README.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
+# Euphony-Speaker
+![euphony_logo](https://user-images.githubusercontent.com/8490058/129358383-14b6f34d-9264-4c49-9538-3ef2e62a6490.png)
```
Comment (yeonns) [inline README.md:2]: How about putting a newline(\n) at the end for readability?
The easiest way is that "Enter a <space> twice at the end of the sentence".
Reference: https://bj25.tistory.com/14

**Review by yeonns — COMMENTED** [review]
Below picture is preview of your README file.
I marked the part where it would be good to put a newline character.

image

**Comment by jopopcorn** [comment]
Oh I see. I'll review it carefully next time. Thanks for telling me!

**Review by yeonns — APPROVED** [review]
Thank you for editing README! Look so nice! 👍 
After @designe review, it will be merged 😄

**Review by designe — COMMENTED** [review]
https://github.com/euphony-io/euphony-speaker/issues/4
In general, for a demo app, this seems to be a very friendly explanation.
It also looks good to put the agreed results in the form of co-contribution after sufficient communication with the team members who work together on the issue #4.

Here are some my comments :)

**Comment by jopopcorn** [comment]
Thank u for your detailed feedback! We'll revise it after we discuss it again.

**Comment by designe** [comment]
> Thank u for your detailed feedback! We'll revise it after we discuss it again.

@euphony-io/coophony 
please self-review here :)

File: README.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+Acoustic Data Telecommunication Library. This is for Android version.  
+Euphony provides a handiness library designed to communicate with other devices(android and web) using mic and recorder.
+
+Official Facebook Page : https://www.facebook.com/euphonyproject  
+Official Library Site : https://dev.jbear.co/euphony
+
+**Euphony Speaker demo app** that generates noiseless sound wave data using frequencies in the inaudible band to communicate with other devices.
```
Comment (sujin9) [inline README.md:10]: It'll be better if this line comes first and adds some explanation more !
And I think this sentence is kinda long, like trying to explain all things in one line,
so how about explaining it in a few simple sentences?

**Comment by aiclaudev** [comment]
> > Thank u for your detailed feedback! We'll revise it after we discuss it again.
> 
> @euphony-io/coophony
> please self-review here :)


# Euphony-Speaker 

euphony_logo

**Euphony Speaker demo app** generates noiseless sound wave to send the data.
When transferring data, the 'Speaker' generates an inaudible frequency and the 'Listener' receives it.

Follow the steps below and experience sound communication !



## Getting Started

### 1. Open Android Studio
Clone the project and open it with Android Studio.

### 2. Run the app
Select an Android emulator or device and run the app.

screenshot</img>

You can enter the text you want to transmit. You can also select the number of transmissions in the spinner.  
You can send or stop sending text via the button at the bottom.

A guidance screencast is available [here](https://www.youtube.com/watch?v=eeg3ShVhSXs).

💡 Recommendation! You can try it with [Euphony-Listener demo app](https://github.com/euphony-io/euphony-listener).



## Compatibility

Minimum Android SDK: Euphony requires at least Android API 14.



## License

Euphony-speaker is licensed under the Apache 2.0 license. See the [LICENSE](https://github.com/euphony-io/euphony-speaker/blob/master/LICENSE) file for details.



## Report Issues

To report a specific problem or feature request, [please open a new issue](https://github.com/euphony-io/euphony-speaker/issues/new).



## Library

This demo-app is based on Euphony Library.

Euphony is Acoustic Data Telecommunication Library. This is for Android version.  
Euphony provides a handiness library designed to communicate with other devices(android and web) using mic and recorder.

Official Facebook Page : https://www.facebook.com/euphonyproject  
Official Library Site : https://dev.jbear.co/euphony

**Comment by aiclaudev** [comment]
I modified the README file by reflecting the mentor's feedback.
Tell me if there's anything missing or if there's anything else that needs to fix.

File: README.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
+# Euphony-Speaker
+![euphony_logo](https://user-images.githubusercontent.com/8490058/129358383-14b6f34d-9264-4c49-9538-3ef2e62a6490.png)
+
+Acoustic Data Telecommunication Library. This is for Android version.
```
Comment (aiclaudev) [inline README.md:4]:
```suggestion
[Acoustic Data Telecommunication Library.](https://github.com/euphony-io) This is for Android version.  
```

**Comment by JuSeongPil** [comment]
I make some change the config for the customer's view
​
# Soundless-check-in
​
euphony_logo
​
This is a check-in application that replace the QR code.
The QR code is hard for adults to using their mobile.
And this soundless check-in application can identifies the visitors following 3 information
-1. There Phone number
-2. There Residential Area
-3.The store that they visit
​
## Reason for making this application
​
The existing 'QR code check-in'  has the following problems.
​
* If the lens of the device receiving the data is dirty, it is not recognized well.
* If the surroundings are too bright or too dark, it is not recognized well.

* The process of accessing QR codes is complicated, especially for the elderly.


But, This application has no such inconvenience as using sound wave communication.


## Run the app



Images or video ...


## Report Issues


To report a specific problem or feature request, [please open a new issue](https://github.com/euphony-io/soundless-check-in/issues/new).


## Library


It is based on Euphony library.

Official Facebook Page : https://www.facebook.com/euphonyproject

Official Library Site : https://dev.jbear.co/euphony







Please comment free:)

**Comment by JuSeongPil** [comment]
@euphony-io/coophony
please review free!

# Euphony-speaker
​
euphony_logo
​

Euphony Speaker demo app generates noiseless sound wave to send the data.
When transferring data, the 'Speaker' generates an inaudible frequency and the 'Listener' receives it.

Follow the steps below and experience sound communication !
Euphony Speaker demo app generates noiseless sound wave to send the data.
When transferring data, the 'Speaker' generates an inaudible frequency and the 'Listener' receives it.

This is a check-in application that replace the QR code.
The QR code is hard for adults to using their mobile.
And this soundless check-in application can identifies the visitors following 3 information
-1. There Phone number
-2. There Residential Area
-3.The store that they visit


Follow the steps below and experience sound communication !
​
## Reason for making this application
​
The existing 'QR code check-in'  has the following problems.
​
* If the lens of the device receiving the data is dirty, it is not recognized well.
* If the surroundings are too bright or too dark, it is not recognized well.

* The process of accessing QR codes is complicated, especially for the elderly.


But, This application has no such inconvenience as using sound wave communication.


## Run the app



Images or video ...


## Report Issues


To report a specific problem or feature request, [please open a new issue](https://github.com/euphony-io/soundless-check-in/issues/new).


## Library


It is based on Euphony library.

Official Facebook Page : https://www.facebook.com/euphonyproject

Official Library Site : https://dev.jbear.co/euphony







Please comment free:)

**Comment by JuSeongPil** [comment]
@euphony-io/coophony

sorry for late.
I added simple principle of speaker app!
please review free!:)

# Euphony-speaker
​
euphony_logo
​
Euphony Speaker demo app generates noiseless sound wave to send the data .
They are make a specific frequency then implement inaudible sonic communication by adjusting the amplitude.

When the 'Speaker' generates an inaudible frequency data by lowing there amplitude but still keep the information
 and the 'Listener' receives it.

Follow the steps below and experience sound communication !
​
## 1. Get start by open android studio
Clone the project and open it with Android Studio.
## 2. Run the app
screenshot</img>
You can enter the text you want to transmit. You can also select the number of transmissions in the spinner.
You can send or stop sending text via the button at the bottom.

A guidance screencast is available here.

💡 Recommendation! You can try it with Euphony-Listener demo app.

## Report Issues


To report a specific problem or feature request, [please open a new issue](https://github.com/euphony-io/soundless-check-in/issues/new).

## Compatibility
Minimum Android SDK : Euphony requires at least Android API 14.
## License
Euphony-speaker is licensed under the Apache 2.0 license. See the LICENSE file for details.
## Report Issues
To report a specific problem or feature request, please open a new issue.
## Library


It is based on Euphony library.

Official Facebook Page : https://www.facebook.com/euphonyproject

Official Library Site : https://dev.jbear.co/euphony







Please comment free:)

**Review by yeonns — COMMENTED** [review]
After the self-review is completed, please work so that only one commit remains in this PR. 😄

**Comment by jopopcorn** [comment]
@designe @yeonns
We updated README again based on feedback. Please review it!

File: README.md [hunk 4]
```diff
@@ -0,0 +1,51 @@
+# Euphony-Speaker
+![euphony_logo](https://user-images.githubusercontent.com/8490058/129358383-14b6f34d-9264-4c49-9538-3ef2e62a6490.png)
```
Comment (designe) [inline README.md:2]: I think this logo would look better on the library header.
How do you think?

**Comment by designe** [comment]
Thanks @sujin9 @aiclaudev @JuSeongPil @jopopcorn for leaving comments.
Is this final version of README for euphony-speaker?

File: README.md (hunk 4 above)
Comment (JuSeongPil) [reply README.md:2]: oh i think so too.
I will update via comment asap !
thank you :)

**Comment by jopopcorn** [comment]
> Thanks @sujin9 @aiclaudev @JuSeongPil @jopopcorn for leaving comments.
> Is this final version of README for euphony-speaker?

That’s right. I’ll revise your feedback and upload it again. thank you!

**Comment by jopopcorn** [comment]
@designe @yeonns 
I uploaded the final version of README, so please check it.
Thank you for taking care of it! 😄

**Review by designe — APPROVED** [review]
Excellent :)
Thanks for your contribution!

**Review by yeonns — APPROVED** [review]
Look so nice 👍

**Pull request merged**